library/Vanilla/Formatting/Formats/HtmlFormat.php

```diff
@@ -185,6 +185,9 @@ public function parseImageUrls(string $content): array {
         /** @var \DOMNode $domImages */
         foreach ($domImages as $domImage) {
             $src = $domImage->getAttribute('src');
+            if (preg_match("/emoji/", $src)) {
```

Comment:
Do we know where in the path the "/emoji" will be? If we can be more specific, I think we should... just to make sure no outside paths have "emoji" in it.

Reply:
@cchabilall83 You should just be able to check the `class` attribute.

Reply:
done